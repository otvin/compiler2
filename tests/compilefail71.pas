program compilefail71(output);
{invalid use of succ}


var r:real;


begin
	r := succ(r);
end.
