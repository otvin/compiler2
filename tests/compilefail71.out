[1mtests/compilefail71.pas:9:10: [0m[1m[91merror:[0m Function succ() requires parameter of ordinal type
     9 | 	r := succ(r);